CMakeLists.txt

```diff
@@ -323,9 +323,9 @@ set(PG_VERSION "${PG_VERSION_MAJOR}.${PG_VERSION_MINOR}")
 # Ensure that PostgreSQL version is supported and consistent with src/compat.h
 # version check
 if((${PG_VERSION_MAJOR} LESS "12")
-   OR (${PG_VERSION_MAJOR} GREATER "14")
+   OR (${PG_VERSION_MAJOR} GREATER "15")
    AND NOT (${EXPERIMENTAL}))
-  message(FATAL_ERROR "TimescaleDB only supports PostgreSQL 12, 13 and 14")
+  message(FATAL_ERROR "TimescaleDB only supports PostgreSQL 12, 13, 14 and 15")
```

Comment:
I dont think this should be part of this patch instead we should make CI run with -DEXPERIMENTAL=ON. We should only enable this once PG15 fully works.